src/v2/providers/alerts/crashlytics.ts

```diff
@@ -117,6 +117,7 @@ export interface NewAnrIssuePayload {
 
 /**
  * A custom CloudEvent for Firebase Alerts (with custom extension attributes).
+ * @typeParam T - the data type for crashlytics alerts that is wrapped in a FirebaseAlertData object.
```

Comment:
Going global with this nit: suggest backticks for `FirebaseAlertData` everywhere it crops up, thanks!